ignite/metrics/nlp/bleu.py

```diff
@@ -97,6 +97,9 @@ class Bleu(Metric):
         device: specifies which device updates are accumulated on. Setting the
             metric's device to be the same as your ``update`` arguments ensures the ``update`` method is
             non-blocking. By default, CPU.
+        average: specifies which type of averaging to use (macro or micro)
+            for more details refer https://www.nltk.org/_modules/nltk/translate/bleu_score.html
+            Default: "macro"
 
     Example:
```

Comment:
Maybe we could add an example comparing the `micro` and `macro` averages in the same way than nltk

https://github.com/nltk/nltk/blob/f989fe65d421e7ea4d1037a00f07eaeee3ad6a29/nltk/translate/bleu_score.py#L136